ext/php8/compatibility.h

```diff
@@ -54,37 +52,13 @@
 #define _GET_UNUSED_MACRO_OF_ARITY(_1, _2, _3, _4, _5, ARITY, ...) UNUSED_##ARITY
 #define UNUSED(...) _GET_UNUSED_MACRO_OF_ARITY(__VA_ARGS__, 5, 4, 3, 2, 1)(__VA_ARGS__)
 
-#if PHP_VERSION_ID < 70000
-#define PHP5_UNUSED(...) UNUSED(__VA_ARGS__)
-#define PHP7_UNUSED(...) /* unused unused */
-#else
 #define PHP5_UNUSED(...) /* unused unused */
 #define PHP7_UNUSED(...) UNUSED(__VA_ARGS__)
```

Comment:
Do we want to remove these as part of this PR or a follow up PR?

Reply:
ops, good catch https://github.com/DataDog/dd-trace-php/pull/1165/commits/534b344505facd27d33abc4ee0a7f5812107dc27

Reply:
From the failed builds it looks like these are still being referenced in the codebase.